FlowCrypt/Controllers/SideMenu/Main/SideMenuNavigationController.swift

```diff
@@ -57,6 +57,8 @@ final class SideMenuNavigationController: ENSideMenuNavigationController {
             $0.delegate = self
             $0.animationDuration = Constants.animationDuration
         }
+        let ekmService = EKMService(appContext: appContext)
```

Comment:
This looks good. I propose to call it `EkmVcHelper`. 

We are calling so many things a `service` in this codebase that the word has lost its meaning (to me, service is something that runs, not a collection of methods).